.gitignore

```diff
@@ -1,10 +1,13 @@
 # Generated by Cargo
 # will have compiled files and executables
-/target/
+**/target/**
```

Comment:
this should not be needed if bazaar is member of the workspace in the root Cargo.toml. But it's fine to leave it